learning_journal/views/default.py

```diff
@@ -25,8 +25,7 @@ def write(request):
         new_title = request.POST["title"]
         new_body = request.POST["body"]
         new_date = time.strftime("%m/%d/%Y")
-        new_id = request.POST['id']
-        new_entry = Entry(title=new_title, body=new_body, creation_date=new_date, id=new_id)
+        new_entry = Entry(title=new_title, body=new_body, creation_date=new_date)
 
         request.dbsession.add(new_entry)
 
```

Comment:
Can't comment below this line, but what you're telling me here is that after you create a new learning journal entry, you want the user to go back to the "make a new learning journal entry" page. Is that really the behavior that you want to encourage?